src/styles/product-card.scss

```diff
@@ -0,0 +1,158 @@
+@mixin center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$card-width: 200px;
+$card-height: 408px;
+
+p {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  padding: 0 16px;
+}
+
+.product {
+  &-picture {
+    @include center;
+    height: 134px;
+    width: 160px;
+    padding-left: 19px;
+    padding-right: 19px;
+    padding-top: 32px;
```

Comment:
You can use shorthand padding property
https://www.w3schools.com/css/css_padding.asp